ngraph/test/runtime/interpreter/evaluates_map.cpp

```diff
@@ -443,17 +472,148 @@ namespace {
+            default:
+                return false;
+        }
+#undef REF_CALL
+    }
+
+    template<element::Type_t ET>
+    bool evaluate(const shared_ptr<op::v0::Convert> &op, const HostTensorVector &outputs,
+                  const HostTensorVector &input) {
+        using T = typename element_type_traits<ET>::value_type;
+#define REF_CALL(U) \
+        runtime::reference::convert<T, typename element_type_traits<U>::value_type>(\
+            input[0]->get_data_ptr<T>(),\
+            outputs[0]->get_data_ptr<U>(),\
+            shape_size(input[0]->get_shape()));
+
+
+        switch (input[0]->get_element_type()) {
+            case element::Type_t::boolean:
+                REF_CALL(element::Type_t::boolean);
+            case element::Type_t::i8:
+                REF_CALL(element::Type_t::i8);
+            case element::Type_t::i16:
+                REF_CALL(element::Type_t::i16);
+            case element::Type_t::i32:
+                REF_CALL(element::Type_t::i32);
+            case element::Type_t::i64:
+                REF_CALL(element::Type_t::i64);
+            case element::Type_t::u8:
+                REF_CALL(element::Type_t::u8);
+            case element::Type_t::u16:
+                REF_CALL(element::Type_t::u16);
+            case element::Type_t::u32:
+                REF_CALL(element::Type_t::u32);
+            case element::Type_t::u64:
+                REF_CALL(element::Type_t::u64);
+            case element::Type_t::f16:
+                REF_CALL(element::Type_t::f16);
+            case element::Type_t::f32:
+                REF_CALL(element::Type_t::f32);
+            case element::Type_t::f64:
+                REF_CALL(element::Type_t::f64);
+            default:
+                return false;
+        }
+#undef REF_CALL
+    }
+
+//    TODO: Rewrite to v1
+    template<element::Type_t ET>
+    bool evaluate(const shared_ptr<op::v1::OneHot> &op, const HostTensorVector &outputs,
+                  const HostTensorVector &inputs) {
+        using T = typename element_type_traits<ET>::value_type;
+        runtime::reference::one_hot<T>(inputs[0]->get_data_ptr<ET>(),
+                                       outputs[0]->get_data_ptr<ET>(),
+                                       inputs[0]->get_shape(),
+                                       outputs[0]->get_shape(),
+                                       op->get_axis());
         return true;
     }
 
+    template<element::Type_t ET>
+    bool evaluate(const shared_ptr<op::v0::RNNCell> &op, const HostTensorVector &outputs,
+                  const HostTensorVector &inputs) {
+//        runtime::reference::rnn_cell(inputs[0]->get_data_ptr<char>(),
+//                                outputs[0]->get_data_ptr<char>(),
+//                                inputs[0]->get_shape(),
+//                                outputs[0]->get_shape(),
+//                                op->get_reduction_axes());
+        return true;
+    }
+
+    //TODO: Rewrite to v1
```

Comment:
Do you mean references have to be updated to support the v1 pad operation? Please add more informative comment